### DISCUSSION TITLE
AWS

### DISCUSSION THREAD

**Discussion by Acogero** [ThallesLazaro/GunboundTH-Server-and-Game-Portable#4] [Ideas]
Você já conseguiu colocar em um EC2 da AWS? Para ficar mais fácil de jogar com os amigos?

**Comment by ThallesLazaro** [comment]
> Você já conseguiu colocar em um EC2 da AWS? Para ficar mais fácil de jogar com os amigos?

Então, neste exato momento estou com o servidor de gunbound em teste em uma maquina linux ubuntu com ip publico, se quiser testar o IP é: 200.98.64.245, percebi que pra jogar em rede ele funciona perfeitamente, mais colocando um IP publico, na hora de sincronizar as salas READY dos player não funciona, nem o chato dentro das salas, estou tentando resolver este problema, minha programação em python ainda e fraca.

Obs: Se quiser testar o servidor com ip publico junto comigo me manda uma mensagem no whatsapp: 31991342357 para combinarmos um horário.

Outra coisa interessante que coloquei no servidor é a parte de logs que pode ser acessada via HTTP: http://200.98.64.245/

**Reply by Acogero** [reply]
Estava tentando por em uma máquina EC2 na AWS, mas ainda não consegui logar no servidor. :/
Já te chamo aí no WhatsApp.
